CitationStyleLanguagePlugin.inc.php

```diff
@@ -337,8 +337,8 @@ public function getCitation($request, $article, $citationStyle = 'apa', $issue =
 			$citationData->author = array();
 			foreach ($authors as $author) {
 				$currentAuthor = new stdClass();
-				$currentAuthor->family = htmlspecialchars($author->getLastName());
-				$currentAuthor->given = htmlspecialchars($author->getFirstName());
+				$currentAuthor->family = htmlspecialchars($author->getLocalizedLastName());
+				$currentAuthor->given = htmlspecialchars($author->getLocalizedFirstName()) .' '. htmlspecialchars($author->getLocalizedMiddleName());
```

Comment:
This will always leave `given` with a trailing space when no middle name is entered. Not sure if CSL trims its input, but IMO it would be better to supply it clean.